js/repo.js

```diff
@@ -0,0 +1,26 @@
+var repos = {};
+
+repos.all = [];
+repos.activity = [];
+
+repos.requestAll = function(callback) {
+  $.ajax({
+    type: 'GET',
+    url: 'https://api.github.com/users/sabbyt/repos?sort=updated',
+    headers: {Authorization: 'token ' + githubToken},
+  }).done(function(data){
+    repos.all = data;
+    //then render in repoView
+  }).done(callback);
+};
```

Comment:
Consider adding a .fail() section to this. Currently your users won't see anything is github is down or something. They will just be wondering why it's empty.